packages/material-ui/src/styles/withStyles.js

```diff
@@ -14,6 +16,7 @@ import createGenerateClassName from './createGenerateClassName';
 import getStylesCreator from './getStylesCreator';
 import getDisplayName from '../utils/getDisplayName';
```

Comment:
Shoudn't all of those now point to `@material-ui/utils`?

Reply:
Yes